src/Miew.js

```diff
@@ -1426,13 +1467,9 @@ Miew.prototype._performAO = (function() {
   // var _kernelOffsets = [-3.0, -2.0, -1.0, 0.0, 1.0, 2.0, 3.0];
   var _kernelOffsets = [-2.0, -1.0, 0.0, 1.0, 2.0];
 
-  return function(srcColorBuffer, srcDepthBuffer, targetBuffer, tempBuffer, tempBuffer1) {
+  return function(srcColorBuffer, srcDepthBuffer, targetBuffer, tempBuffer, tempBuffer1, normalsBuffer) {
```

Comment:
I think, it'll be great to rename srcDepthBuffer->srcDepthTexture. Also we should replace function params in the order: {color, normal, depth, target, temp, temp2} and rename  normalsBuffer->normalBuffer to maintain naming consistency.